tests/test-dirs/completion/infix.t/run.t

```diff
@@ -1,4 +1,4 @@
-  $ $MERLIN single complete-prefix -position 11:10 -prefix "Z." \
+  $ $MERLIN single complete-prefix -position 9:10 -prefix "Z." \
```

Comment:
This is a drive by - while performing this refactor, I noticed that this position was incorrect. Leaving this as it was would still result in the test passing